src/renderer/styles/statetable.less

```diff
@@ -55,3 +55,49 @@
 .StateTable-Warning-Info {
   padding-right: 5rem;
 }
+
+.fileDisplay {
+  height: 100%;
+  min-height: 120px;
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+}
+
+#matchDisplay {
+  height: 100%;
+  min-height: 120px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.fileHeaderStyle {
+  font-size: 18px;
+  font-weight: bold;
+  margin: 0;
+}
+
+.errorMessage {
+  color: red;
+  font-weight: bold;
+  font-size: 18px;
```

Comment:
issue: prefer `rem` across all font sizes for zoom scaling purposes https://www.joshwcomeau.com/css/surprising-truth-about-pixels-and-accessibility/#accessibility-considerations-5